inc/sainexthopgroup.h

```diff
@@ -41,11 +41,45 @@ typedef enum _sai_next_hop_group_type_t
     /** Next hop group is ECMP */
     SAI_NEXT_HOP_GROUP_TYPE_ECMP,
 
+    /** Next hop protection group.  Contains primary and backup next hops. */
+    SAI_NEXT_HOP_GROUP_TYPE_PROTECTION,
+
     /* Other types of next hop group to be defined in the future, e.g., WCMP */
 
 } sai_next_hop_group_type_t;
 
 /**
+ * @brief No switchover triggers supported
+ */
+#define SAI_NEXT_HOP_SWITCHOVER_TRIGGER_NULL                         0x00000000
```

Comment:
As discussed in the call last week, the control plane will no longer provide these flags but identifiers for the port and BFD session.